Code/Scripts/Evidence.gd

```diff
@@ -54,27 +56,32 @@ func missing_evidence(ghost):
 func possible_outcomes():
 	var evidence_text = ""
 	for ghost in ghosts:
-		if (ghost[1] & evidence) == evidence:
-			if items_selected == 3:
-				evidence_text = "Ghost is a " + ghost[0] + "\n"
-			elif items_selected == 0:
+		if ghost[1] == evidence:
+			evidence_text = "Ghost is a " + ghost[0] + "\n"
+		elif (ghost[1] & evidence) == evidence and not (ghost[1] & exclude) != 0:
+			if items_selected == 0:
 				evidence_text = "All" + "\n"
 			else:
 				evidence_text += ghost[0] + missing_evidence(ghost[1]) + "\n"
 	return evidence_text
 
 func _on_Evidence_List_multi_selected(index, selected):
-	if get_node("Evidence_List").get_item_custom_bg_color(index) == Color(0, 1, 0, 1):
+	if get_node("Evidence_List").get_item_custom_bg_color(index) == Color(1, 0, 0, 1):
 		get_node("Evidence_List").set_item_custom_bg_color(index, Color(0, 0, 0, 1))
+		excluded_items.erase(index)
+		exclude -= index_to_evidence[index]
+		get_node("Evidence_Outcomes").text = possible_outcomes()
+	elif get_node("Evidence_List").get_item_custom_bg_color(index) == Color(0, 1, 0, 1):
+		get_node("Evidence_List").set_item_custom_bg_color(index, Color(1, 0, 0, 1))
 		items_selected -= 1
-		current_items.erase(index)
+		excluded_items.append(index)
+		exclude += index_to_evidence[index]
 		evidence -= index_to_evidence[index]
 		get_node("Evidence_Outcomes").text = possible_outcomes()
-	else:
-		if items_selected <= 2:
```

Comment:
I don't know why this bugged me. They are logically equivalent, but `< 3` always feels a bit more concrete than `<= 2`.